client/src/panels/panel_declarations/services/single_service_panels/service_overview_v2.js

```diff
@@ -0,0 +1,157 @@
+                  <ProgressGauge
+                    value={
+                      _.countBy(flat_standard_reports, "is_target_met").true ||
+                      0
+                    }
+                    total_value={flat_standard_reports.length}
+                  />
+                </div>
+              </dd>
+            </Fragment>
+          )}
+          <dt>
+            {`${text_maker("identification_methods")} (${
+              most_recent_report.year
+            })`}
+          </dt>
+          <dd>
+            <FancyUL className="service_overview-fancy-ul">
+              {[
+                <div key="uses_cra_as_identifier" className="identifier-item">
+                  <TM style={{ lineHeight: 2 }} k="uses_cra_as_identifier" />
+                  <div>
+                    {
+                      available_icons[
+                        available_keys[
+                          most_recent_report.cra_business_ids_collected
+                        ]
+                      ]
+                    }
+                    <TM
+                      style={{ marginLeft: 5 }}
+                      k={
+                        available_keys[
+                          most_recent_report.cra_business_ids_collected
+                        ]
+                      }
+                    />
+                  </div>
+                </div>,
+                <div key="uses_sin_as_identifier" className="identifier-item">
+                  <TM style={{ lineHeight: 2 }} k="uses_sin_as_identifier" />
+                  <div>
+                    {
+                      available_icons[
+                        available_keys[most_recent_report.SIN_collected]
+                      ]
+                    }
+                    <TM
+                      style={{ marginLeft: 5 }}
+                      k={available_keys[most_recent_report.SIN_collected]}
+                    />
+                  </div>
+                </div>,
+              ]}
+            </FancyUL>
+          </dd>
+          <dt>{text_maker("related_programs")}</dt>
+          <dd>
+            {_.map(service.program_ids, (program_id) => {
+              const program = Subject.Program.lookup(program_id);
+              return (
+                program && (
+                  <p key={program_id}>
+                    <a href={infograph_href_template(program)}>
+                      {program.name}
+                    </a>
+                  </p>
+                )
+              );
+            })}
+          </dd>
+          <dt>{text_maker("services_fees")}</dt>
+          <dd>
+            <TM
+              k={
+                service.collects_fees
+                  ? "does_charge_fees"
+                  : "does_not_charge_fees"
+              }
+            />
+          </dd>
+          <dt>{text_maker("service_link_text")}</dt>
+          <dd>
+            {!_.isEmpty(service.urls) &&
+              (service.urls.length === 1 ? (
```

Comment:
Oh since it doesn't cost much space now in this format, you should get rid of this check